newrelic/resource_newrelic_alert_policy_channel.go

```diff
@@ -136,3 +158,19 @@ func policyChannelExists(client *newrelic.NewRelic, policyID int, channelID int)
 
 	return false, nil
 }
+
+func policyChannelsExist(client *newrelic.NewRelic, policyID int, channelIDs []int) (bool, error) {
+	for _, id := range channelIDs {
+		channelExists, err := policyChannelExists(client, policyID, id)
```

Comment:
Can we batch this so we don't have to make a request each time for each iteration?

Reply:
Refactored to only make one request instead of multiple.